internal/store/lease.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package store
+
+import (
+	coordinationv1 "k8s.io/api/coordination/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+
+	"k8s.io/kube-state-metrics/pkg/metric"
+	generator "k8s.io/kube-state-metrics/pkg/metric_generator"
+)
+
+var (
+	descLeaseLabelsDefaultLabels = []string{"lease"}
+
+	leaseMetricFamilies = []generator.FamilyGenerator{
+		{
+			Name: "kube_lease_owner",
+			Type: metric.Gauge,
+			Help: "Information about the Lease's owner.",
+			GenerateFunc: wrapLeaseFunc(func(l *coordinationv1.Lease) *metric.Family {
+				labelKeys := []string{"owner_kind", "owner_name"}
+
+				owners := l.GetOwnerReferences()
+				if len(owners) == 0 {
+					return &metric.Family{
+						Metrics: []*metric.Metric{
+							{
+								LabelKeys:   labelKeys,
+								LabelValues: []string{"none", "none"},
```

Comment:
It seems like we did this slightly different in other collectors, like the pod collector.

https://github.com/kubernetes/kube-state-metrics/blob/e881e4ab55a3dacaf47ffb54e260cb36fb416883/internal/store/pod.go#L121-L163

Can we make sure this is consistent?

Reply:
Fixed the label's value.